src/app/demo/demo.component.html

```diff
@@ -209,11 +228,21 @@
   </div>
 
   <div class="fullscreen">
-    <div>
+    <main #main>
       <router-outlet></router-outlet>
-    </div>
-    <footer>
-      This demo uses angular-archwizard version 7.0.0-develop
+    </main>
+
+    <footer class="bg-light text-center py-3 mt-4">
+      <p>This demo uses angular-archwizard version 6.0.0</p>
+
+      <form class="form mx-auto github text-center">
+        <a class="github-button"
+           href="https://github.com/madoar/angular-archwizard"
+           data-icon="mark-github"
+           data-size="large"
+           data-show-count="true"
+           aria-label="Visit madoar/angular-archwizard on GitHub">Github</a>
+      </form>
```

Comment:
I know I'm nitpicking a bit here but would it be possible to reduce the spaces between the different elements a bit? The whole footer now seems to take quite a lot of space compared to what the demo is meant to show: a number of example use cases for `angular-archwizard`

Reply:
I reduced the paddings and margins of the footer and I think it looks fine. :D
but feel free to tell me if you want to reduce it more :D.